web/src/components/organisms/Asset/Asset/hooks.ts

```diff
@@ -3,16 +3,24 @@ import { useCallback, useEffect, useMemo, useState } from "react";
 import Notification from "@reearth-cms/components/atoms/Notification";
 import { Asset, PreviewType } from "@reearth-cms/components/molecules/Asset/asset.type";
 import { viewerRef } from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/Asset";
+import {
+  fileFormats,
+  imageFormats,
+  compressedFileFormats,
+} from "@reearth-cms/components/molecules/Common/Asset";
 import {
   Asset as GQLAsset,
   PreviewType as GQLPreviewType,
   useGetAssetQuery,
   useUpdateAssetMutation,
 } from "@reearth-cms/gql/graphql-client-api";
 import { useT } from "@reearth-cms/i18n";
+import { getExtension } from "@reearth-cms/utils/file";
 
 import { convertAsset } from "../convertAsset";
 
+export type ViewerType = "TileSet" | "Image" | "SVG" | "Unsupported";
```

Comment:
I think these are used code-side only, right? Since they aren't shown to the user or matched with the same values from backend values, etc, you don't need to capitalize. Keeping simple makes typos less likely.
```
export type ViewerType = "tileset" | "image" | "svg" | "unsupported";
```

Reply:
Also, I don't think unsupported is needed. It is just a fallback

Reply:
Done!

Now that I am thinking about it I renamed "tileset" to "cesium" since it is more general and the viewer supports 3dtilesets and other formats like "czml" and "kml"... 

Also I think that I will keep "unsupported" since it is used here:
https://github.com/reearth/reearth-cms/blob/999e8dbe034d1f7350fbabb68ad1ed199277c6b0/web/src/components/molecules/Asset/Asset/AssetBody/previewToolbar.tsx#L28

Please let me know what do you think.